Formatting Tags and Ordered Alphabetically

rglc12 opened the pull request:
In response to Issue #2236 - Improve filter labels (project tags):

I have modified all tags in the listOfProjects.js file so that the are capitalised and added a sort() function to the projectList array in the CardContainer.jsx file to order the tags alphabetically. 
Also added my name to the Contributors.md file.

rglc12 committed:
Formatted Tags and Ordered Alphabetically

Modified all tags so that the are capitalised and added a sort()
function to the array in the CardContainer.jsx file to order the tags
alphabetically. Also added my name to the Contributors.md file.

rglc12 committed:
Merge branch 'master' into fixing-label-organisation

Roshanjossey commented on app/src/components/ProjectList/CardsContainer.jsx:
```diff
@@ -14,6 +14,7 @@ export default class CardsContainer extends React.Component {
     }
     this.setTags = new Set();
     this.filterOptions = [];
+    projectList.sort();
```
It's not projectList we wanna sort. We should sort filterOptions.

Roshanjossey commented on app/src/components/ProjectList/listOfProjects.js:
```diff
@@ -176,56 +176,57 @@ const projectList = [
 ];
 
 export default projectList;
+export default projectList;
```
You should only use one default export per module

Roshanjossey reviewed (changes requested):
Hi @rglc12, Thank you for your contribution. I've shared my thoughts as inline comments. Please address those comments

rglc12 committed:
Addressing @Roshanjossey Inline Comment

I have removed the second export in the listOfProjects.js file and changed the sort() function to target the filterOptions array in CardsContainer.jsx.

rglc12 committed:
Merge branch 'master' into fixing-label-organisation

rglc12 commented:
Hey @Roshanjossey, I have addressed your inline comments and made the modification to my contribution. Thanks!

sara-02 commented on app/src/components/ProjectList/listOfProjects.js:
```diff
@@ -176,55 +176,55 @@ const projectList = [
     githubLink: 'https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue',
     description: 'VS Code is a new type of tool that combines the simplicity of a code editor with what developers need for their core edit-build-debug cycle.',
-    tags: ['typescript', 'text-editor', 'electron']
+    tags: ['Typescript', 'Text-Editor', 'Electron']
```
`TypeScript`

sara-02 commented on app/src/components/ProjectList/listOfProjects.js:
```diff
@@ -99,41 +99,41 @@ const projectList = [
-    description: 'Parse, validate, manipulate, and display dates in javascript',
-    tags: ['javascript', 'frontend', 'meta'],
+    description: 'Parse, validate, manipulate, and display dates in JavaScript',
+    tags: ['JavaScript', 'Front-end', 'Meta'],
```
`Front-End` lets keep the capitalization consistent.

sara-02 commented on app/src/components/ProjectList/listOfProjects.js:
```diff
@@ -68,19 +68,19 @@ const projectList = [
     githubLink: 'https://github.com/vuejs/vue/issues?q=is%3Aopen+is%3Aissue+label%3A%22contribution+welcome%22',
     description: 'A progressive, incrementally-adoptable JavaScript framework for building UI on the web',
-    tags: ['javascript', 'UI', 'frontend'],
+    tags: ['JavaScript', 'UI', 'Front-end'],
```
`Front-End`

sara-02 commented on app/src/components/ProjectList/listOfProjects.js:
```diff
@@ -4,57 +4,57 @@ const projectList = [
       'https://github.com/habitat-sh/habitat/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3AE-Easy',
     description: 'Modern applications with built-in automation',
-    tags: ['docs','frontend','rust','multiOS'],
+    tags: ['Docs','Front-end','Rust','MultiOS'],
```
`Front-End`

Roshanjossey commented on app/src/components/ProjectList/CardsContainer.jsx:
```diff
@@ -14,6 +14,7 @@ export default class CardsContainer extends React.Component {
     }
     this.setTags = new Set();
     this.filterOptions = [];
+    filterOptions.sort();
```
@rglc12, This is still not correct. Here you're trying to sort it after filterOptions is defined. You gotta sort it after filterOptions is populated

Roshanjossey commented:
@rglc12, We've requested for more changes. Please go through them.

Also make sure you verify changes locally in development mode before committing and pushing changes.

To do this, go to app directory, run `yarn` to install all dependencies, and `yarn start` to start development server to view changes.

You can learn more about this from https://github.com/Roshanjossey/first-contributions/blob/master/app/README.md

rglc12 committed:
Fixed Ordering of Tags + Capitalization

The ordering of tags on each of project has been fixed and are ordered alphabetically. I have also capitalized the tags that were missed.

rglc12 committed:
Merge branch 'fixing-label-organisation' of https://github.com/rglc12/first-contributions into fixing-label-organisation

rglc12 committed:
Merge branch 'master' into fixing-label-organisation

rglc12 commented:
Hey @Roshanjossey, apologies about the second commit. I have now done testing locally and the tags are now ordered in alphabetical order. @sara-02 Thank you for pointing out where I missed out some tags in the listOfProjects.js file, I have also corrected them.

sara-02 commented:
LGTM, @Roshanjossey PTAL and merge it.

rglc12 committed:
Merge branch 'master' into fixing-label-organisation

Roshanjossey commented:
Thank you for fixing this @rglc12
